specification.json

```diff
@@ -848,6 +848,27 @@
             "content": "`PROVIDER_READY` handlers attached after the provider is already in a ready state MUST run immediately.",
             "RFC 2119 keyword": "MUST",
             "children": []
+        },
+        {
+            "id": "Requirement 5.3.4",
+            "machine_id": "requirement_5_3_4",
+            "content": "When the provider's `on context changed`  is called, `PROVIDER_STALE` handlers MUST run.",
```

Comment:
```suggestion
            "content": "When the provider's `on context changed` is called, `PROVIDER_STALE` handlers MUST run.",
```

Reply:
@Kavindu-Dodan this is autogenerated, so please also update the source.